### PR TITLE
adding owlbot and notebook ci

### LINKED ISSUE #7809: Add notebook sample CI testing template from synthtool
## In which file did you encounter the issue?
This requires adding two files at project root: `owlbot.py` and `repo-metadata.json`

### Did you change the file? If so, how?
Two new files are added 

## Describe the issue
Currently, notebooks in this repo do not have CI testing. In order to import testing pipeline files from synthtool, this repo must be configured to subscribe to `owlbot` updates.

### PR DESCRIPTION
**Pull request by loferris**
## Description

Fixes #7809

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Review by parthea — APPROVED** [review]

File: .github/snippet-bot.yml [hunk 1]
```diff
@@ -1,4 +0,0 @@
-aggregateChecks: true
-alwaysCreateStatusCheck: true
-ignoreFiles:
-  - README.md
```
Comment (parthea) [inline .github/snippet-bot.yml:4]: I don't think this file should be deleted.

**Review by parthea — CHANGES_REQUESTED** [review]
Need to check why owl bot is deleting `.github/snippet-bot.yml`

**Review by parthea — APPROVED** [review]

**Review by parthea — CHANGES_REQUESTED** [review]
The `header-check` is failing with this message 
```
.cloud-build/CheckPythonVersion.py should have a copyright year of 2022
.cloud-build/cleanup/cleanup-cloudbuild.yaml is missing a valid license header.
.cloud-build/cleanup/requirements.txt is missing a valid license header.
.cloud-build/execute_changed_notebooks_cli.py should have a copyright year of 2022
.cloud-build/execute_notebook_cli.py should have a copyright year of 2022
.cloud-build/execute_notebook_helper.py should have a copyright year of 2022
.cloud-build/execute_notebook_remote.py should have a copyright year of 2022
.cloud-build/notebook-execution-test-cloudbuild-single.yaml is missing a valid license header.
.cloud-build/notebook-execution-test-cloudbuild.yaml is missing a valid license header.
.cloud-build/requirements.txt is missing a valid license header.
.cloud-build/test_folders.txt is missing a valid license header.
.cloud-build/utils/__init__.py is missing a valid license header.
.github/workflows/ci.yaml is missing a valid license header.
.github/workflows/linter/requirements.txt is missing a valid license header.
.github/workflows/linter/run_linter.sh should have a copyright year of 2022
```

**Comment by dandhlee** [comment]
Hey @loferris, friendly ping on this!

**Comment by loferris** [comment]
> The `header-check` is failing with this message
> 
> ```
> .cloud-build/CheckPythonVersion.py should have a copyright year of 2022
> .cloud-build/cleanup/cleanup-cloudbuild.yaml is missing a valid license header.
> .cloud-build/cleanup/requirements.txt is missing a valid license header.
> .cloud-build/execute_changed_notebooks_cli.py should have a copyright year of 2022
> .cloud-build/execute_notebook_cli.py should have a copyright year of 2022
> .cloud-build/execute_notebook_helper.py should have a copyright year of 2022
> .cloud-build/execute_notebook_remote.py should have a copyright year of 2022
> .cloud-build/notebook-execution-test-cloudbuild-single.yaml is missing a valid license header.
> .cloud-build/notebook-execution-test-cloudbuild.yaml is missing a valid license header.
> .cloud-build/requirements.txt is missing a valid license header.
> .cloud-build/test_folders.txt is missing a valid license header.
> .cloud-build/utils/__init__.py is missing a valid license header.
> .github/workflows/ci.yaml is missing a valid license header.
> .github/workflows/linter/requirements.txt is missing a valid license header.
> .github/workflows/linter/run_linter.sh should have a copyright year of 2022
> ```

Cannot be fixed manually on this PR, as the headers are missing in the synthtool template, so owlbot overwrites.

**Comment by kweinmeister** [comment]
Closing this PR, as #10011 will supersede it.

**Pull request closed without merging**